### PR TITLE
Add SQL Bindings Extension to insiders gallery

### PR DESCRIPTION
**Pull request by VasuBhog**
This PR adds SQL Bindings extension to the insider gallery (sql-database-projects is updated in insiders as well).

Build for VSIX (using RC1 build):
https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=149954&view=artifacts&pathAsName=false&type=publishedArtifacts

### CODE REVIEW

**Comment by VasuBhog** [comment]
Tested this locally with the insider's edition of sql-database-projects (0.16.1) as well (both in vscode and ADS).

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -4686,6 +4686,63 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "91",
+					"extensionName": "sql-binding",
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json:4692]: sql-bindings

Comment (Charles-Gagnon) [reply extensionsGallery-insider.json]: Nevermind, I just went and fixed the issues

**Comment by VasuBhog** [comment]
Thanks @Charles-Gagnon!

**Pull request merged**